src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({
+  person: { name, age, sex, isMarried, partnerName },
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+    {age && <p className="Person__age">{`I am ${age}`}</p>}
+
+    <p className="Person__partner">
+      {isMarried
+        ? `${partnerName} is my ${sex === 'f' ? 'husband' : 'wife'}`
```

Comment:
Nested ternary looks complicated, let's create a separate variable and use it here

`const partner = sex === 'f' ? 'husband' : 'wife'`